test/source/tests/gmail.ts

```diff
@@ -24,10 +24,21 @@ export type GmailCategory = 'inbox' | 'sent' | 'drafts' | 'spam' | 'trash';
 export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: TestWithBrowser) => {
 
   if (testVariant === 'CONSUMER-LIVE-GMAIL') {
-
-    const pageHasSecureReplyContainer = async (t: AvaContext, browser: BrowserHandle, gmailPage: ControllablePage, { isReplyPromptAccepted }: { isReplyPromptAccepted?: boolean } = {}) => {
+    const pageHasSecureReplyContainer = async (
+      t: AvaContext,
+      browser: BrowserHandle,
+      gmailPage: ControllablePage,
+      {
+        isReplyPromptAccepted,
+        composeCount
+      }:
+      {
+        isReplyPromptAccepted?: boolean,
+        composeCount?: number
+      } = {}
+    ) => {
       const urls = await gmailPage.getFramesUrls(['/chrome/elements/compose.htm'], { sleep: 0 });
-      expect(urls.length).to.equal(1);
+      expect(urls.length).to.equal(composeCount ?? 1);
       if (typeof isReplyPromptAccepted !== 'undefined') {
         const replyBox = await browser.newPage(t, urls[0]);
```

Comment:
To be frank, I also agree with you.
I think original author's thought was like this.
Will follow your way.

> Is this because in our particular test the "secure reply compose container" is located higher than the other "reply or forward buttons" container